storage.py

```diff
@@ -17,6 +17,14 @@ def remove(self):
 
     def set(self):
         pass
-    
-    def add(self):
-        pass
\ No newline at end of file
+
+    def add(self, key, value):   #Катя
+        key = key
+        value = value
+
+        if key in self.data.keys():
+            raise Exception
+        else:
+            d = {key : value}
+            self.data.update(d)
+            print(self.data)
```

Comment:
Print statement is good for debugging but is redundant in final version (this line can be removed).

Reply:
I respect print so much I always have a couple of them in my code

Reply:
By the way, if you respect print, it's a good idea to reimplement _repr()_ method of class Storage. It should return a string that holds a printable representation of an object.
If it's not reimplemented, you get something like that:

```
>>> st = Storage({"one": 1, "two": 2})
>>> print(st)
>>> <storage.Storage object at 0x7f9ea8013150>
```